frontend/src/components/PaginatedTable.js

```diff
@@ -37,31 +37,23 @@ function PaginatedTable ({
   const filtersRef = useRef(filters);
   const sortByRef = useRef(sortBy);
   const [currentPage, setCurrentPage] = useState(1);
-  const nextPage = currentPage + 1;
-  const nextPageEndIndex = nextPage * pageSize;
 
   const startIndex = (currentPage - 1) * pageSize;
   const endIndex   = Math.min((currentPage) * pageSize, totalCount);
 
-  const isLastPage = endIndex >= totalCount;
-
   const dataSource =
     items.slice(startIndex, endIndex)
       .map(id => itemsByID[id]);
 
   const hasUnloadedItems = dataSource.some(d => d === undefined);
   const isCurrentPageUnloaded = ((endIndex - 1) > items.length) || hasUnloadedItems;
-  const doesNextPageContainUnloaded = !isLastPage && nextPageEndIndex > items.length && items.length < totalCount;
-  const shouldLoadNextChunk =
-    !loading && (isCurrentPageUnloaded || doesNextPageContainUnloaded);
+  const shouldLoadNextChunk = !loading && isCurrentPageUnloaded;
 
   if (shouldLoadNextChunk) {
     let offset
 
     if (isCurrentPageUnloaded)
       offset = Math.floor(startIndex / pageSize) * pageSize;
-    else if (doesNextPageContainUnloaded)
```

Comment:
The value of `offset` should not be left undefined if `isCurrentPageUnloaded` is false.